master/buildbot/test/fake/fakedb.py

```diff
@@ -1655,7 +1656,7 @@ def setState(self, objectid, name, value):
 
     def atomicCreateState(self, objectid, name, thd_create_callback):
         value = thd_create_callback()
-        self.states[objectid][name] = json.dumps(value)
+        self.states[objectid][name] = json.dumps(bytes2NativeString(value))
```

Comment:
I know this is nearly done, lets keep calm and continue working cleanly.

I needed to change only one place to fix the test_www. see:	aba82ca